groovy-language-server/src/main/java/com/palantir/groovylanguageserver/GroovycWrapper.java

```diff
@@ -116,6 +131,61 @@ public Path getWorkspaceRoot() {
     }
 
     @Override
+    public Map<String, Set<SymbolInformation>> getReferences() {
+        return references;
```

Comment:
We do compile as soon as a file is opened. But I agree it would be strange that you could not search for symbols unless you opened a file. Finding referenxss on the other hand doesn't matter as much since its based on user interaction with an open file. I think we should consider starting to compile from a new thread in initialize. This is something for a separate PR though.